### PR TITLE
Cybersun's blacksite ruin

### PR DESCRIPTION
**Pull request by Marabout2**
## About The Pull Request

Apparently, there is a need for more ruin so here it is, one additional ruin to the rooster.

## Why It's Good For The Game

More ruins mean more things to be explored ?
Screenshots
With areas :
image

Without any areas :
image

Ingame :
image

## Changelog

:cl:
add: Added a new Cybersun related ruin
code: Modified the map_catalogue.txt file; the icemoon.dm areas ruin file; modified the icemoon.dm datum to add said ruin.
:cl:

### CODE REVIEW

File: code/datums/ruins/icemoon.dm [hunk 1]
```diff
@@ -64,3 +69,5 @@
 	id = "crashed_holemaker"
 	description = "Safety records for early Nanotrasen Spaceworks vessels were, and always have been, top of their class. Absolutely no multi-billion credit projects have been painstakingly erased from history. (Citation Needed)"
 	suffix = "icemoon_crashed_holemaker.dmm"
+
+
```
Comment (thgvr) [inline code/datums/ruins/icemoon.dm:73]: one too many newlines at end of file

File: code/game/area/areas/ruins/icemoon.dm [hunk 2]
```diff
@@ -40,3 +40,31 @@
...
+/area/ruin/powered/hangar
+	name = "Hangar"
+	icon_state = "storage"
+
+/area/ruin/powered/maintenance
+	name = "Maintenance"
+	ambientsounds = MAINTENANCE
+	lighting_colour_tube = "#ffe5cb"
+	lighting_colour_bulb = "#ffdbb4"
+	sound_environment = SOUND_AREA_TUNNEL_ENCLOSED
```
Comment (thgvr) [inline code/game/area/areas/ruins/icemoon.dm:54]:
```suggestion
	sound_environment = SOUND_AREA_TUNNEL_ENCLOSED
	
```

File: code/datums/ruins/icemoon.dm (hunk 1 above)
Comment (thgvr) [inline code/datums/ruins/icemoon.dm]:
```suggestion
```

**Comment by tiramisuapimancer** [comment]
chicken typo

**Comment by Latentish** [comment]
Add 3 pictures to show your ruin in your description. 1 in game screenshot (Use the command "Change View Range" to get one while ghosted), and two SDMM screenshots (one with areas on, one with only area borders on) (make sure to use the screenshot button in sdmm).

**Comment by Marabout2** [comment]
Alright, should be done with the images too !

**Comment by Latentish** [comment]
image
Don't copy paste content from another ruin for this please.

**Review by Latentish — CHANGES_REQUESTED** [review]
image
Don't run pipes through walls unless it is for an the singular exhaust pipe of a ship or compound.

image
Add a dangerous mob here =)

Add some fluff incriminating the syndies here as fringe ex members, or deserters.

Change your shuttle here, it's lifted straight from the bombed out airbase.

I think this is a strong ruin, fix your stuff up and we can look at TMing it.

**Comment by Latentish** [comment]
image
This door is facing the wrong direction. you have a diagonal wall here (don't do that), you missed one underwall pipe, and these lights need to have a wire node under them to turn on.
Aside from this I'm very very impressed.

**Comment by Latentish** [comment]
image
this rainbow area is certainly concerning . I would recommend looking at the li-tieguai and using the red decals that it uses if these ones are being a bit too 70's.

**Comment by meemofcourse** [comment]
happy april fools lat

**Review by Latentish — CHANGES_REQUESTED** [review]
image
Hi!!!! This is going to cause an atmosphere difference. I'd recommend closing it up and adding an airlock, or making icemoon outside subytpes for cement and stairs.

image
Open plating actually already has an icemoon subtype. Use that instead.

Other than that this looks pretty good. Get those tiles fixed and we can TM this.

**Review by goober3 — CHANGES_REQUESTED** [review]
please swap out the snow turfs outside for the /lit variant. also, fix your unrotated airlocks.

**Comment by Latentish** [comment]
I want this merged already. Get this passing checks, and I will give it a lookover on local, and I'll merge it.

**Comment by SomeguyManperson** [comment]
syndie mobs in this need to be swapped to the new hostile human paths

File: _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm [hunk 3]
```diff
@@ -0,0 +1,6964 @@
...
+/obj/structure/cable{
+	icon_state = "4-8"
+	},
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/hallway)
+"aW" = (
+/obj/structure/chair{
+	dir = 4
+	},
+/mob/living/simple_animal/hostile/syndicate/melee/space,
```
Comment (SomeguyManperson) [inline _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm:48]:
```suggestion
/mob/living/simple_animal/hostile/human/syndicate/melee/space,
```

File: _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm [hunk 4]
```diff
@@ -0,0 +1,6964 @@
...
+/obj/effect/turf_decal/corner/opaque/bar/border,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/hallway)
+"ou" = (
+/obj/effect/mob_spawn/human/corpse,
+/obj/effect/decal/cleanable/blood/gibs/down,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/testing_facilities)
+"oA" = (
+/mob/living/simple_animal/hostile/syndicate/ranged/space,
```
Comment (SomeguyManperson) [inline _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm:946]:
```suggestion
/mob/living/simple_animal/hostile/human/syndicate/ranged/space,
```

File: _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm [hunk 5]
```diff
@@ -0,0 +1,6964 @@
...
+"EE" = (
+/obj/effect/decal/cleanable/blood/old,
+/obj/machinery/light/dim/directional/west,
+/obj/effect/turf_decal/corner/opaque/purple/border{
+	dir = 8
+	},
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/testing_facilities)
+"EH" = (
+/mob/living/simple_animal/hostile/syndicate/melee/sword,
```
Comment (SomeguyManperson) [inline _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm:1802]:
```suggestion
/mob/living/simple_animal/hostile/human/syndicate/melee/sword,
```

File: _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm [hunk 6]
```diff
@@ -0,0 +1,6964 @@
...
+/obj/machinery/atmospherics/pipe/manifold/supply/hidden/layer2{
+	dir = 4
+	},
+/turf/open/floor/mineral/plastitanium/red,
+/area/ruin/powered/crew_facilities)
+"KL" = (
+/obj/structure/chair/office/dark{
+	dir = 4
+	},
+/mob/living/simple_animal/hostile/syndicate/ranged/space/stormtrooper{
```
Comment (SomeguyManperson) [inline _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm:2189]:
```suggestion
/mob/living/simple_animal/hostile/human/syndicate/ranged/space/stormtrooper{
```

File: _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm [hunk 7]
```diff
@@ -0,0 +1,6964 @@
...
+	dir = 4
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/hallway)
+"Mp" = (
+/obj/structure/chair/office{
+	dir = 1;
+	name = "Pilot's Chair"
+	},
+/mob/living/simple_animal/hostile/syndicate/ranged/smg/pilot{
```
Comment (SomeguyManperson) [inline _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm:2276]:
```suggestion
/mob/living/simple_animal/hostile/human/syndicate/ranged/smg{
```
pilot subtype was killed brutally in ruincut

File: _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm [hunk 8]
```diff
@@ -0,0 +1,6964 @@
...
+	dir = 8
+	},
+/obj/machinery/power/apc/auto_name/directional/east,
+/obj/structure/cable,
+/obj/item/bedsheet/orange,
+/turf/open/floor/carpet/royalblack,
+/area/ruin/powered/crew_facilities)
+"XQ" = (
+/obj/machinery/power/apc/auto_name/directional/north,
+/mob/living/simple_animal/hostile/syndicate/ranged/smg/orion,
```
Comment (SomeguyManperson) [inline _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm:3045]:
```suggestion
/mob/living/simple_animal/hostile/human/syndicate/ranged/smg/orion,
```

File: _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm [hunk 9]
```diff
@@ -0,0 +1,6964 @@
...
+	},
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 4
+	},
+/turf/open/floor/concrete/reinforced{
+	light_range = 2
+	},
+/area/ruin/powered/hangar)
+"Ur" = (
+/mob/living/simple_animal/hostile/syndicate/melee,
```
Comment (SomeguyManperson) [inline _maps/RandomRuins/IceRuins/icemoon_cybersun_blacksite.dmm:2856]:
```suggestion
/mob/living/simple_animal/hostile/human/syndicate/melee,
```

**Review by SomeguyManperson — COMMENTED** [review]
should fix present issues

**Comment by rye-rice** [comment]
sowwy, nwo prwogwess iwn a mwonth, rweopwen wwen wrorking on iwt agwian

**Pull request closed without merging**